src/main/kotlin/screeps/api/Constants.kt

```diff
@@ -289,3 +307,37 @@ val TEXT_ALIGN_CENTER: TextAlignConstant = "center".unsafeCast<TextAlignConstant
 val TEXT_ALIGN_RIGHT: TextAlignConstant = "right".unsafeCast<TextAlignConstant>()
 val ALGORITHM_ASTAR: AlgorithmConstant = "astar".unsafeCast<AlgorithmConstant>()
 val ALGORITHM_DIJKSTRA: AlgorithmConstant = "dijkstra".unsafeCast<AlgorithmConstant>()
+
+
+external interface PowerClassConstant : StringConstant
+external interface PowerEffectConstant : IntConstant
+
+external interface POWER_CLASS {
```

Comment:
This interface should be Camelcase `PowerClass`
Also it has on usages anywhere so I assume you forgot to add a property somewhere, is that correct?

Reply:
No. The interface should actually be POWER_CLASS :disappointed: Check https://docs.screeps.com/api/#Constants

Reply:
Better to keep the same names as in the official doc. I am not very pleased about he type in question, but I guess it's how it's gonna be for now

Reply:
The _type name_ should be camelcase, which is consistent with all of the type names in the library. The _value name_ should be the same as in the docs:

```suggestion
external interface PowerClass {
    val OPERATOR: PowerClassConstant
}
external val POWER_CLASS: PowerClass
```

Reply:
Oh woops, I agree of course :sweat_smile: baby brain